src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -7,16 +7,25 @@
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    using Orleans.Providers;
+
+    public class OrleansJsonSerializer : IExternalSerializer
     {
+        public const string UseFullAssemblyNamesProperty = "UseFullAssemblyNames";
+        public const string IndentJsonProperty = "IndentJSON";
         private static JsonSerializerSettings defaultSettings;
         private Logger logger;
 
+        static OrleansJsonSerializer()
+        {
+            defaultSettings = GetDefaultSerializerSettings();
+        }
```

Comment:
I think the defaultSettings cached value is an old thing and we do not use it anymore. I'm not suggesting we should use it, because mutating this shared object is not expected but possible (and was causing sneaky side-effects, so that's why we stopped using it).